packages/routing/src/analytics.ts

```diff
@@ -1,3 +1,9 @@
-import { route } from 'typesafe-routes';
+import { route, stringParser } from 'typesafe-routes';
 
-export default route('/analytics', {}, {});
+const metric = route('/:metric', { metric: stringParser }, {});
```

Comment:
Not sure if this is a issue, it would be better to confirm with Tiff or João, but the path in network currently is `interest-groups` and `working-groups`.

<img width="394" alt="Screenshot 2024-04-03 at 11 36 19" src="https://github.com/yldio/asap-hub/assets/16595804/2f4a3d80-6332-4cfd-b51d-13939e51e036">
<img width="402" alt="Screenshot 2024-04-03 at 11 36 12" src="https://github.com/yldio/asap-hub/assets/16595804/f641437e-3fea-438e-a6e9-b3a9d484bf10">


but here in analytics is `interestGroup` and `workingGroup`. This difference between camel case and kebab case bothers me 😅 but I'm not sure if it's an actual issue